tools/stable-tag.sh

```diff
@@ -0,0 +1,147 @@
+#!/usr/bin/env bash
+
+set -eo pipefail
+
+BASE=$(cd $(dirname "${BASH_SOURCE[0]}")/.. && pwd)
```

Comment:
Someday I should add `shellcheck` as a linter. In the mean time,
```suggestion
BASE=$(cd "$(dirname "${BASH_SOURCE[0]}")/.." && pwd)
```